python/cugraph/centrality/katz_centrality_wrapper.pyx

```diff
@@ -85,6 +85,11 @@ def katz_centrality(input_graph, alpha=0.1, max_iter=100, tol=1.0e-5, nstart=Non
     c_katz.katz_centrality(g, &c_katz_centrality_col, alpha, max_iter, tol, has_guess, normalized)
 
     if input_graph.renumbered:
-        df['vertex']=input_graph.edgelist.renumber_map[df['vertex']]
+        if isinstance(input_graph.edgelist.renumber_map, cudf.DataFrame):
+            unrenumered_df = df.merge(input_graph.edgelist.renumber_map, left_on='vertex', right_on='id', how='left').drop(['id', 'vertex'])
```

Comment:
line is to long

Reply:
Added an utility function to be called in for most of the bindings for unrenumbering in new commit. However a few bindings need different code within repeating codeblock. That could be simplified in consequent PRs addressing #678 as code could possibly change.